cgmes/cgmes-model/src/main/java/com/powsybl/cgmes/model/AbstractCgmesModel.java

```diff
@@ -23,8 +26,10 @@
  */
 public abstract class AbstractCgmesModel implements CgmesModel {
 
-    protected AbstractCgmesModel() {
+    protected AbstractCgmesModel(Reporter reporter) {
+        // FIXME(Luma) we must remove properties from here. They are not used!
         this.properties = new Properties();
+        this.reporter = reporter;
```

Comment:
I'd rather not retain the reporter in an object, especially if there's only one call (or few calls) on it. You'd rather put the reporter as an argument of CgmesModel::read, I find it's easier to see what's going on with it that way... at the cost of polluting a bit the API.

A good compromise is what you did in another part of that PR, save it in an existing context object which is passed along. Unfortunately here there's no such context object.